app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t("title")
+    if page_title.empty?
```

Comment:
em dùng toán tử ba ngôi (`ternary operator`) để viết lại block code `if...else...end` này nhé